docs/package_guide.md

```diff
@@ -0,0 +1,141 @@
+increased as you make changes to this file or update the used commit to change other files that are not directly tied to your program such as configs or icons for launchers.
+
+`package/your-packagename/package`
+```
+# vim: set ft=sh:
+pkgname=your-package-name
+pkgdesc="A short human readable description of what your package is and/or does"
+url=https://github.com/username/your-package-repo
+pkgver=0.1.0-1
+# Time of updating this file in UTC (in 24h format)
+timestamp=yyyy-mm-ddThh:mmZ
+# A rough section like utils, remarkable-apps (if in launchers), launchers, math, etc.
+section=utils
+maintainer="Your Name <your_mail@domain.tld>"
+# Your license. Examples are MIT, GPL-3.0-or-later, Apache-2.0,  etc.
+license=your_license_type
+# Optional. Include this to ensure those packages get installed (separated by spaces)
+depends=depending-packagename-a depending-packagename-b
+# Include this line if your package cant coexist with another one
+conflicts=conflicting-packagename
+
+# Which docker image to use. See https://github.com/toltec-dev/toolchain/
+# The images are debian based and allow you to install additional packages with apt.
+# Examples (version may be out of date): base:v1.1, qt:v1.1, python:v1.1, rust:v1.1
+image=base:v1.1
+source=(
+    https://github.com/username/your-package-repo/archive/FULL_SHA1_OF_COMMIT.zip
+)
+sha256sums=(
+    SHA256_OF_THE_FILE_ABOVE
+)
+
+build() {
+    # Commands to compile your source
+    # You will be in the directory of your specified file in "source="
+    # (including that file itself)
+    # e.g. make
+}
+
+package() {
+    # Commands to copy your finished files into the directory
+    # that will represent the reMarkable root (/) where those
+    # are going to be installed to.
+    # You can use simple commands like cp or ln but the
+    # command "install" with proper permissions is preferred
+    # (`man install` for more or https://linux.die.net/man/1/install)
+
+    # The following environment variables are to be used
+    # "$srcdir" - The directory you where in when build() ran
+    # "$pkgdir" - The final folder containing the root where your files should go
+    # "$recipedir" - `package/your-packagename/`. Contains your package file and others
+
+    # Examples
+    # Add a single config file or non-executable resource
+    # install -D -m 644 "$srcdir"/default_config "$pkgdir"/opt/etc/your_package.conf
+    #
+    # Add a empty directory
+    # install -d "$pkgdir"/opt/etc/draft/icons
+    #
+    # Add a executable (binary/script)
+    # install -D -m 755 "$srcdir"/build/programname "$pkgdir"/opt/bin/programname
+}
+```
+
+#### Adding hooks
+
+Should you need to run one or more scripts on certain points you can also add the following scripts (which should be executable and have a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)))
+
+**NOTE:** This part will probably be changed a bit in the future. See https://github.com/toltec-dev/toltec/issues/60 for more.
```

Comment:
This is now outdated since #65 was merged. See https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#install-section-optional for updated docs.